mobility_games/games/wall_audio_player.py

```diff
@@ -98,35 +105,32 @@ def run(self):
+                    """ MAKE THIS A PART OF THE UPDATE OF THE POSITION INSTEAD OF RECALCULATING ALL OF THIS FOR EVERY SOUND PLAYED!!!!!"""
+                    if (self.pitch): #if pitch modulation
+                        multiplier = self.maxfreq/math.pow(2, (self.maxfreqdist-self.altsounddist)) #calculate multiplier based on maxfreq, maxfreqdist, and altsounddist
                         freq = max(min(multiplier*math.pow((abs(self.dist+random.random()*self.randrange)-self.altsounddist), 2)+65.4*2, self.maxfreq), 65.4*2)*2 #set frequency based on distance
-
-                        #if self.dist < .7:
-                        #    freq = ac.quantize(freq, quantizetype = "M", key="C")
-                        #elif self.dist < 1.4:
-                        #    freq = ac.quantize(freq, quantizetype = "dom7", key="G")
-                        #else:
-                        freq = ac.quantize(freq, quantizetype = self.quantizetype, key=self.key)
-                        #print (freq):
+                        freq = ac.quantize(freq, quantizetype = self.quantizetype, key=self.key) #quantize audio
                     else:
-                        if (self.pitchdefault != ''):
+                        try: #try to set default pitch
                             freq = ac.freq(self.pitchdefault)
-                    if (self.vol):
-                        vol = (.8/(1+math.exp((self.dist)/1.5-1))+.2)*vol#insert volume function here. that goes from 1 to 0 with increasing distance to wall.
+                        except: #if that doesn't work, do nothing, leaving the pitch as whatever it previously was.
+                            pass
+                    if (self.vol): #if volume modulation
+                        vol = (.8/(1+math.exp((self.dist)/1.5-1))+.2)*vol #insert volume function here. that goes from 1 to 0 with increasing distance to wall.
                     else:
-                        vol = self.voldefault
+                        try: #try to set default volume
+                            vol = self.voldefault
+                        except: #if that doesn't work, do nothing, leaving the volume as whatever it previously was.
+                            pass
                     #self.synth = ac.delay(self.synth, 1, self.s*.2, .5) #add delay to the synth
 
-                    if self.track:
-                        pass
-                    else:
+                    if not self.track: #if no track
                         synth = ac.synth(freq, synth="sin", fadebool=False) #create synth for sound
                         synth = ac.fade_out(ac.fade_in(synth, fadeinendtime, fadeintime), fadeoutendtime, fadeouttime)
                         ac.playstream(synth, self.player, seconds = seconds, volume = vol)#synth, self.player, seconds = .5, volume = vol)
+                    else:
+                        """ Set Up Music Here"""
```

Comment:
Is this a todo for set up music here?  If you do not have plans to fill in the body of the else statement, I'd just remove it.